src/App.js

```diff
@@ -1,13 +1,48 @@
-import React from 'react';
+import React from "react";
+import Form from "./components/TodoComponents/TodoForm";
+import List from "./components/TodoComponents/TodoList";
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todoList: [
+        {
+          task: "Organize Garage",
+          id: 1528817077286,
+          completed: false
+        },
+        {
+          task: "Bake Cookies",
+          id: 1528817084358,
+          completed: false
+        }
+      ],
+      newTodo: ""
+    };
+  }
+
+  changeHandler = event => {
+    console.log(event.target.value);
+    this.setState({ newTodo: event.target.value });
+  };
+
+  addNewTodo = event => {
+    event.preventDefault();
+    this.setState({
+      todoList: [...this.state.todoList, { task: this.state.newTodo }],
```

Comment:
it would make sense to add `id` and `completed` properties in addition to a `task` property to the todo object. more on this on Thursday I'm sure.